sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml

```diff
@@ -0,0 +1,92 @@
+---
+parent: 'common/test_cases/minimal.yml'
+extensions:
+  actions:
+    - type: "host_script"
+      title: "Copy GCE credentials into AWS instance"
+      script: |-
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.config/gcloud/descheduler-credentials.json openshiftdevel:/data/gce-credentials.json
+    - type: "script"
+      title: "Setup GCE authentication"
+      script: |-
+        sudo tee -a /etc/yum.repos.d/google-cloud-sdk.repo << EOM
+        [google-cloud-sdk]
+        name=Google Cloud SDK
+        baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64
+        enabled=1
+        gpgcheck=1
+        repo_gpgcheck=1
+        gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+               https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+        EOM
+
+        sudo yum install -y google-cloud-sdk uuid
+        gcloud auth activate-service-account --key-file /data/gce-credentials.json
+        gcloud config set project openshift-descheduler-infra
+        gcloud config set compute/zone us-east1-b
+    - type: "script"
+      title: "Build descheduler"
+      script: |-
+        export GOPATH=/data
+        export PATH=$GOPATH:$PATH
+        cd $GOPATH/src/github.com/kubernetes-incubator/descheduler
+        hack/verify-gofmt.sh
+        make build
+    - type: "script"
+      title: "Provision GCE cluster"
+      script: |-
+        cd /data/src/github.com/kubernetes-incubator/descheduler
+        hack/e2e-gce/gcloud_create_cluster.sh
+    - type: "script"
+      title: "run descheduler tests"
+      script: |-
+        export GOPATH=/data
+        export PATH=$GOPATH:$PATH
+        cd $GOPATH/src/github.com/kubernetes-incubator/descheduler
+        make test-e2e
+  system_journals:
+    - systemd-journald.service
+overrides:
+  post_actions:
```

Comment:
Well yeah. I don't see any other way. We need to change the GCE project cause we switch to `openshift-descheduler-infra` and if anything goes wrong we need to make sure the GCE project is back to `openshift-gce-devel`. And we need to collect artefacts before the GCE output is assembled. Right now we don't generated any artefacts but later on we will.

Reply:
Not the worst thing in the world, this will probably work.